keyless/pepper/service/src/lib.rs

```diff
@@ -211,14 +233,19 @@ fn process_common(
         uid_val,
         aud: final_aud,
     };
-    info!(
-        session_id = session_id,
-        iss = input.iss,
-        aud = input.aud,
-        uid_val = input.uid_val,
-        uid_key = input.uid_key,
-        "PepperInput is available."
-    );
+
+    if !aud_overridden {
```

Comment:
I guess if the `override_aud` is set, no need to log it...

Reply:
which is what's being done here?